concent-vm/helper-scripts/concent-migrate.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash -e
+
+source concent-env.sh
+cd concent_api/
+
+django_databases_name=(
+    control
+    storage
+)
```

Comment:
This a list. You should name it in plural. E.g. `django_database_names` or `django_databases`.

Reply:
Done